### ISSUE TITLE
namespace?

### ISSUE THREAD

**Issue by MikeBinstock** [bvandenbos/redis_feature_control#3]
is there a way to setup a namespace?

**Comment by bvandenbos** [comment]
Not currently, though it would be a welcome patch.  Not sure that I'll get to it in the next few weeks.

**Comment by bvandenbos** [comment]
I lied.  Released 0.0.2 just now.  It adds `connection=` which takes a Redis instance which can be wrapped in an Redis::Namespace.  So, if you want a custom namespace do this:

```
redis = Redis.new(:host => host, :post => port)
redis_namespaced = Redis::NameSpace(:some_custom_namespace, redis)
Redis::FeatureControl.redis = redis_namespaced
```

Make sense?

**Comment by MikeBinstock** [comment]
That's great! :) Thanks! :)
